src/CreateHandler/CreateHandlerCommand.php

```diff
@@ -59,9 +59,13 @@ class. For a path, it matches the class namespace against PSR-4 autoloader
      */
     protected function configure() : void
     {
+        $this->handlerArgument = (bool) preg_match('/^action:/', $this->getName())
+            ? 'action'
+            : 'handler'; // default argument
```

Comment:
If we add the property maybe there we can have the default value and change it here only:
```php
if (strpos($this->getName(), 'action:') === 0) {
    $this->handlerArgument = 'action';
}
```